server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -26,11 +30,24 @@ namespace irods {
              itr != plugin_list.end();
              ++itr)
         {
+
+            // Get filepath
+            const auto filepath = fmt::format("{}lib{}.so", plugin_home, *itr);
+
+            // Get checksum
+            char checksum[NAME_LEN];
```

Comment:
Consider adding empty curly braces to the array (i.e. `char checksum[NAME_LEN]{}`).

This helps protect against non-null-terminated strings in the future. Especially if `chksumLocFile()` isn't guaranteed to always produce null-terminated strings.